views/layouts/default.lavender

```diff
@@ -0,0 +1,79 @@
+!!!
+
+head
+  title
+    block title
+  meta(
+    name='keywords',
+    content='Lavender, Lavender programming language,lavenderlang, lavender-lang'
+  )
+  meta(
+    name='description',
+    content='A templating engine for PHP, designed to make your life easier.'
+  )
+  meta(name='viewport', content='width=device-width, initial-scale=1')
+  -appleTouchIconSizes = [57,60,72,76,114,120,144,152,180]
+  for appleTouchIconSize in appleTouchIconSizes
+    link(rel="apple-touch-icon", sizes="#{appleTouchIconSize}x#{appleTouchIconSize}", href="/img/favicons/apple-touch-icon-#{appleTouchIconSize}x#{appleTouchIconSize}.png")
+  -faviconSizes = [16,32,96,192,194]
+  for faviconSize in faviconSizes
+    link(rel="icon", sizes="#{faviconSize}x#{faviconSize}", type="image/png", href="/img/favicons/favicon-#{faviconSize}x#{faviconSize}.png")
+  link(rel="manifest", href="/img/favicons/manifest.json")
+  meta(name="msapplication-TileColor", content="#cdb4cc")
+  meta(name="msapplication-TileImage", content="/img/favicons/mstile-144x144.png")
+  meta(name="theme-color", content="#796878")
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Oswald:400,700')
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Lato:300,700')
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Special+Elite')
+  link(rel='stylesheet', href='http://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css')
+  link(rel='stylesheet', href='http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css')
+  link(rel='stylesheet', href='css/main.css')
+  script(src='https://code.jquery.com/jquery-1.11.2.min.js')
+  script(src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js")
+  script(src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js")
+body
+  if active_class_index
```

Comment:
i'm not totally sure what you're doing here, but since `active_class_index` is only set from one template i'm betting whatever this is would be better done with a block